### PR TITLE
Responsive header sidebar and homepage

### PR DESCRIPTION
**Pull request by IanKBovard**
responsive layout for homepage, header, and footer. menu added for mobile as well.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,11 +2,28 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { Grommet } from 'grommet';
 import { hpe } from 'grommet-theme-hpe';
+import { deepMerge } from 'grommet/utils';
 import { Home, Community, Arcade, StickerWall, Schedule } from './pages/index';
 
+const customHpe = deepMerge(hpe, {
```
Comment (britt6612) [inline src/App.js:8]: This is exactly what I was thinking changes the values!

File: src/components/Header/index.js [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
-            fit="contain"
-            size="small"
-            src="https://us-central1-grommet-designer.cloudfunctions.net/images/lozzi-hpe-com/developer-logo.png"
-          />
-        </Box>
-        <Box align="center" justify="center" direction="row" gap="xsmall">
-          <Text weight="bold">HPE</Text>
-          <Text>Developer</Text>
-        </Box>
+    <HeaderGrommet justify="between" align="center">
```
Comment (britt6612) [inline src/components/Header/index.js:25]:
```suggestion
    <HeaderGrommet pad="medium" justify="between" align="center">
```

File: src/components/Header/index.js [hunk 3]
```diff
@@ -1,38 +1,53 @@
...
+          {layer && (
+            <Layer>
+              <Box pad={{ top: 'xlarge', right: 'large' }}>
+                <Box direction="row" align="center" justify="end">
+                  <Text color="#FFFFFF">CLOSE</Text>
+                  <Button icon={<Close />} onClick={() => setLayer(false)} />
+                </Box>
+                <Box align="start" gap="large" pad="xlarge">
+                  <SideNav />
+                  <ButtonSplit to="https://developer.hpe.com">
```
Comment (reddypramod85) [inline src/components/Header/index.js:42]: Should we remove Register for Discover from the mobile view?

File: src/components/Header/index.js [hunk 4]
```diff
@@ -1,38 +1,53 @@
...
+import React, { useContext, useState, useEffect } from 'react';
+import {
+  Box,
+  Button,
+  Text,
+  Header as HeaderGrommet,
+  Image,
+  ResponsiveContext,
+  Layer,
+  Form,
```
Comment (reddypramod85) [inline src/components/Header/index.js:10]: remove unused import Form

File: src/components/Header/index.js (hunk 3 above)
Comment (IanKBovard) [reply src/components/Header/index.js:41]: its in the figma, just the link to needs to be updated

File: src/components/Header/index.js (hunk 4 above)
Comment (IanKBovard) [reply src/components/Header/index.js]: ill clean up the code in a separate branch

**Pull request merged**